Library/Homebrew/brew.rb

```diff
@@ -141,9 +141,14 @@ class MissingEnvironmentVariables < RuntimeError; end
       end
       tap_commands += %W[#{HOMEBREW_BREW_FILE} tap #{possible_tap.name}]
       safe_system(*tap_commands)
+      ohai "Automatically tapped #{possible_tap.name}. To remove, run `brew untap #{possible_tap.name}`"
     end
 
-    exec HOMEBREW_BREW_FILE, cmd, *ARGV
+    if help_flag
```

Comment:
Could also consider adding `--help` to `ARGV` (or another splatted array) if it's set to avoid a bit of minor duplication?

Reply:
👍 Good idea